CHANGES.md

```diff
@@ -3,7 +3,11 @@ Change Log
 
 ### 1.11 - 2015-07-01
 
+* Breaking changes
+  * Removed `Camera.clone` that was deprecated in 1.10.
 * Deprecated
+  * `CatmullRomSpline` has been deprecated and will be removed in Cesium 1.12.
```

Comment:
Maybe we should rethink this: https://groups.google.com/forum/#!topic/cesium-dev/MMIdPKFNR2s

Deprecating `HermiteSpline` is still OK though.